src/Athena/Controllers/api/InstitutionController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Athena.Core.Repositories;
+using Athena.Core.Models;
+using Athena.Exceptions;
+using System.Net;
+using Athena.Extensions;
+
+namespace Athena.Controllers.api
+{
+    [Route("api/v1/[Controller]")]
+    public class InstitutionController : Controller
+    {
+        private readonly IInstitutionRepository _institutions;
+        private readonly IProgramRepository _programs;
+
+        public InstitutionController(IInstitutionRepository institutionRepository, ICampusRepository campusRepository, IStudentRepository studentRepository, IProgramRepository programsRepository)
+        {
+            _institutions = institutionRepository ?? throw new ArgumentNullException(nameof(institutionRepository));
+            
+            _programs = programsRepository ?? throw new ArgumentNullException(nameof(programsRepository));
```

Comment:
You can get rid of this after moving the one route I mentioned below to cleanup the constructor.